### DISCUSSION TITLE
Use dev for ESP32 installations?

### DISCUSSION THREAD

**Discussion by olddawgpowers** [Dilbert66/esphome-dsckeybus#79] [Q&A]
Thank you for the excellent project. A few months ago I used "new" for my installation with a ESP32.  Now that it has been merged into "dev" moving forward should all ESP32 installs and updates use the "dev" code?     My last update was 12/26/2022 from "new" and everything is working great.  Is there any reason to update to the new files in "dev"?   As a side note in the installation instructions "copy both the dscalarm.yaml file and the subdirectory "dscKeybusInterface" should be changed to copy both the dscalarm.yaml file and the subdirectory "scr"

**Comment by Dilbert66** [comment]
Yes, I do need to update the documentation. I'm playing with adding support for platformio library inclusions so needed a dir restructure. Yes, the src directory replaces dsckeybusinterface.  You don't need to update if all is working fine.  I have an older rev dsc system that needed some adjusments with the firmware.  Also added a bit more functionality to the virtual keypad functions.  Nothing too major.
I'm also adding a Tasmota version so also needed some tweaks.
